### DISCUSSION TITLE
Possible to change foreground color of PWD_COLOR?

### DISCUSSION THREAD

**Discussion by derekcollison** [zimfw/zimfw#484] [Q&A]
I want this to be white if possible. I am sure it's possible. Any pointers helpful. Thanks.

**Comment by ericbn** [comment]
If you're using the eriner or steeef prompts, which can be customized with this environment variable, then add `PWD_COLOR=white` to your .zshrc file. In the eriner prompt though, it sets the background, not the foreground color.

For any prompt in general, check the README.md file on how to customize it. If the customization you're looking for is not described there, then I suggest you [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the module into your own GitHub user and change the code to your needs. Then replace the `zmodule` to include your username first, e.g. `zmodule derekcollison/eriner` replacing `zmodule eriner`.

**Comment by derekcollison** [comment]
Do you know if eriner supports setting the foreground color?

**Reply by ericbn** [reply]
It currently does not support setting the foreground color. The README.md says, under [Settings](https://github.com/zimfw/eriner/blob/master/README.md#settings):
> The background color for each segment can be customized with an environment variable.

**Comment by derekcollison** [comment]
Thx
